### PR TITLE
Ghosts can look at a paper bundle's current page

### PR DESCRIPTION
**Pull request by harryob**
## About The Pull Request

Small QoL fix picked up from #ideas_guys, just shows ghosts the current page in a paper stack.

## Why It's Good For The Game

Ghosts wanna see what page people are looking at in a bundle

## Changelog

:cl:
qol: Observers can now look at the current page of a paper bundle.
/:cl:

### CODE REVIEW

**Comment by harryob** [comment]
man, forgot i didn't build this uncommitted code from a few days back. one mo

File: code/modules/paperwork/paper_bundle.dm [hunk 1]
```diff
@@ -79,15 +79,15 @@
...
+	if(in_range(user, src) || istype(user, /mob/dead/observer))
 		src.attack_self(user)
 	else
 		to_chat(user, SPAN_NOTICE("It is too far away."))
 
 /obj/item/paper_bundle/attack_self(mob/user)
 	..()
 
-	if(!ishuman(user))
+	if(!(ishuman(user) || istype(user, /mob/dead/observer)))
```
Comment (stanalbatross) [inline code/modules/paperwork/paper_bundle.dm:90]:
```suggestion
	if(!(ishuman(user) || isobserver(user)))
```

File: code/modules/paperwork/paper_bundle.dm [hunk 2]
```diff
@@ -79,15 +79,15 @@
 
 /obj/item/paper_bundle/examine(mob/user)
 	to_chat(usr, desc)
-	if(in_range(user, src))
+	if(in_range(user, src) || istype(user, /mob/dead/observer))
```
Comment (stanalbatross) [inline code/modules/paperwork/paper_bundle.dm:82]:
```suggestion
	if(in_range(user, src) || isobserver(user))
```

File: code/modules/paperwork/paper_bundle.dm [hunk 3]
```diff
@@ -170,8 +170,10 @@
 
 		src.attack_self(src.loc)
 		updateUsrDialog()
+	else if(istype(usr, /mob/dead/observer))
```
Comment (stanalbatross) [inline code/modules/paperwork/paper_bundle.dm:173]:
```suggestion
	else if(isobserver(usr))
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**